support-frontend/assets/helpers/subscriptionsForms/guestCheckout.js

```diff
@@ -1,12 +1,12 @@
 // @flow
 
-import type { CheckoutState } from 'helpers/subscriptionsForms/subscriptionCheckoutReducer';
+import type { CheckoutState, WithDeliveryCheckoutState } from 'helpers/subscriptionsForms/subscriptionCheckoutReducer';
 import { getUserTypeFromIdentity } from 'helpers/identityApis';
 import type { UserTypeFromIdentityResponse } from 'helpers/identityApis';
 import { setUserTypeFromIdentityResponse } from 'helpers/subscriptionsForms/formActions';
 
 export const checkIfEmailHasPassword = (email: string) =>
```

Comment:
I realise it's existing, but it's not clear what this function does as it returns void, it looks more like `eventuallySaveUserTypeInRedux` or something to that effect?

Reply:
Yes it is a bit confusing, I'll have a look at refactoring